android/src/paper/java/com/horcrux/svg/NativeSvgViewModuleSpec.java

```diff
@@ -22,7 +20,8 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-public abstract class NativeSvgViewModuleSpec extends ReactContextBaseJavaModule implements TurboModule {
+public abstract class NativeSvgViewModuleSpec extends ReactContextBaseJavaModule
```

Comment:
Why do we have does prettier changes in the PR? I would remove them since they are not in the scope of it.

Reply:
restored.